sdk/include/psdk/winuser.h

```diff
@@ -4923,6 +4923,14 @@ BOOL WINAPI GetWindowInfo(_In_ HWND, _Inout_ PWINDOWINFO);
 BOOL WINAPI GetMonitorInfoA(_In_ HMONITOR, _Inout_ LPMONITORINFO);
 BOOL WINAPI GetMonitorInfoW(_In_ HMONITOR, _Inout_ LPMONITORINFO);
 
+#if (_WIN32_WINNT >= 0x0600)
+#define USER_DEFAULT_SCREEN_DPI 96
+#endif /*_WIN32_WINNT >= 0x0600*/
+
+#if (_WIN32_WINNT >= 0x0605)
```

Comment:
What is `0x0605`?

Reply:
`0x0605` should correspond to Windows 10 Build 1607 (Anniversary Update.) https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getdpiforwindow

Reply:
```
 114 #define NTDDI_WINBLUE                       0x06030000      // Windows 8.1
 115 #define NTDDI_WINTHRESHOLD                  0x0A000000      // Windows 10.0.10240 / 1507 / Threshold 1
 116 #define NTDDI_WIN10                         0x0A000000
 117 #define NTDDI_WIN10_TH2                     0x0A000001      // Windows 10.0.10586 / 1511 / Threshold 2
 118 #define NTDDI_WIN10_RS1                     0x0A000002      // Windows 10.0.14393 / 1607 / Redstone 1
```
and
```
  30 #define _WIN32_WINNT_WINBLUE                0x0603
  31 #define _WIN32_WINNT_WINTHRESHOLD           0x0A00
  32 #define _WIN32_WINNT_WIN10                  0x0A00
```
so they are already using 0x0A0* types of values for the `_WIN32_WINNT` versions for windows 10.

Reply:
🤔Guess the documentation Microsoft provides is wrong there, but this is the same value the `winuser.h` header from the Windows SDK guards against.

Reply:
Otherwise, with the `DLL_EXPORT_VERSION` thing :thinking:

Reply:
> `0x0605` should correspond to Windows 10 Build 1607 (Anniversary Update.)

Early Insider builds were 6.4 or 6.5 but the major version switched to 10.0 long before Windows 10 RTM but there might be features added during this early phase that might be guarded by 0x605 I guess.